vispy/glsl/arrowheads/arrowheads.vert

```diff
@@ -75,10 +75,9 @@ void main (void)
     v_color       = color;
     v_linewidth   = linewidth;
 
-    vec2 body = v2 - v1;
+    vec3 body = $transform(vec4(v2 - v1, 0.)).xyz;
     v_orientation = (body / length(body));
-    v_orientation.y = -v_orientation.y;
```

Comment:
This is not needed anymore because we use the transformation of the camera. The example I proposed at the beginning of this PR :

![vispy canvas_004](https://user-images.githubusercontent.com/15892073/32173260-a8d5b272-bd7e-11e7-8b21-ad825be9ec6d.png)

The [arrows.py](https://github.com/vispy/vispy/blob/master/examples/basics/visuals/arrows.py) example :

![arrows example_003](https://user-images.githubusercontent.com/15892073/32173296-caed7f20-bd7e-11e7-9232-518931f0d787.png)

**Note** : I add an `arrow_color` input parameter which by default has the same color as the line. I didn't change the color in the example but may be I should set `arrow_color='w'` ? If we conserve this arrow color input, it should be tested (such as the definition of arrows along the z-axis)

Reply:
So you are saying the example image above has gray arrows because you didn't set the color in the example code?

And sounds good for the removed line of code above.

Reply:
Exactly, arrows are gray because this is the default color and I didn't change it in the example. This default color is the same as for the line.

Reply:
Ah I see now. What if the arrow_color defaulted to `None` and then `if arrow_color is None: arrow_color = line_color`? So the arrow ends up having the same color as the line by default.

Reply:
Exactly. It was my first intention to use the same color as the line. This is the most natural solution. But if the user define different colors for several lines using an array or a list of colors , there's a different number of colors compare to the number of arrows.

See my example bellow : 

```python
"""To define the traditional XYZ axis, the position of the line is defined using 6 vertices. So 6 colors are needed. For the three arrows, only three colors are needed.
"""
pos = np.array([[0., 0., 0.], [1., 0., 0.],
                [0., 0., 0.], [0., 1., 0.],
                [0., 0., 0.], [0., 0., 1.],
                ])
arrows = np.c_[pos[0::2], pos[1::2]]

line_color = ['red', 'red', 'green', 'green', 'blue', 'blue']
arrow_color = ['red', 'green', 'blue']
```

This is the reason why I chose to dissociate the `color` and `arrow_color` inputs.

Reply:
Good point. Bottom line is the arrows can exist without an associated line, technically.

Reply:
You mean if it's possible to define the arrow-head without the line?